ob_v2p1/ob-examples.md

```diff
@@ -0,0 +1,514 @@
+PublicKey  | <code>https://example.org/publicKey.json</code> | A <code>CryptographicKey</code> document that links to the issuer <code>Profile</code> with its <code>owner</code> property. [See below](#CryptographicKey.
+
+
+<h3 id="CryptographicKey">CryptographicKey Example</h3>
+
+(<a data-cite="OB-SPEC-21#CryptographicKey">definition</a>)
+
+A public key document should describe an Open Badges issuer's public key. For maximum compatibility, it should have its own HTTP(S) identifier, and should identify its issuer using the <code>owner</code> property. The <code>publicKeyPem</code> shown below has been abbreviated with <code>...</code> for readability.
+
+<pre>
+{
+  "@context": "https://w3id.org/openbadges/v2",
+  "type": "CryptographicKey",
+  "id": "https://example.org/publicKey.json",
+  "owner": "https://example.org/organization.json",
+  "publicKeyPem": "-----BEGIN PUBLIC KEY-----\\nMIIBG0BA...OClDQAB\\n-----END PUBLIC KEY-----\\n"
+}
+</pre>
+
+
+<h3 id="RevocationList">RevocationList Example</h3>
+
+(<a data-cite="OB-SPEC-21#RevocationList">definition</a>)
+
+Issuers may have a RevocationList if they use <code>SignedBadge</code> verification. Checking this list is intended as part of the verification process for signed badges, just as checking for the hosted assertion is part of verifying a hosted badge. It is published as a JSON-LD document with type <code>RevocationList</code>. RevocationLists are linked from an issuer Profile via the <code>revocationList</code> property. The RevocationList identifies its issuer with the <code>issuer</code> property.
+
+RevocationLists may identify revoked Assertions through their <code>revokedAssertions</code> property. Individual assertions are identified either by their <code>id</code> or (legacy) <code>uid</code> properties. <code>id</code>-identified Assertions may appear in a RevocationList as that id string or as an object with an <code>id</code> property and other metadata, usually just a <code>revocationReason</code>. The below example shows <code>id</code>s in the <code>urn:uuid</code> namespace, which is a recommended namespace for signed Assertions that do not have a hosted (HTTP) <code>id</code>.
+<pre>
+{
+  "@context": "https://w3id.org/openbadges/v2",
+  "id": "https://example.org/revocationList.json",
+  "type": "RevocationList",
+  "issuer": "https://example.org/issuer",
+  "revokedAssertions": [
+    "urn:uuid:3c574c87-b96f-4f06-8eb5-68a29335b60e",
+    {
+      "id": "urn:uuid:e79a6c18-787e-4868-8e65-e6a4530fb418",
+      "revocationReason": "Honor code violation"
+    },
+    {
+      "uid": "abc123",
+      "revocationReason": "Issued in error."
+    }
+  ]
+}
+</pre>
+
+
+<h3 id="HostedBadge">Revoked Hosted Assertion Example</h3>
+
+(see more about <a data-cite="OB-SPEC-21#HostedBadge">Hosted Verification</a>)
```

Comment:
It turns out the link resolver refuses to generate a link if the target is not an absolute URL. I'll put the absolute URLs in localBiblio.js. That will render the links, but the links will be broken because the documents have not be published yet.